UnitTests/GitUITests/Theming/AppColorDefaultsTests.cs

```diff
@@ -23,12 +25,24 @@ public void Default_values_are_defined_in_AppColorDefaults()
         [Test]
         public void Default_values_are_specified_in_invariant_theme()
         {
-            var controller = new FormThemeEditorController(null, new ThemePersistence());
-            var invariantTheme = controller.LoadInvariantTheme(quiet: true);
-            foreach (AppColor name in Enum.GetValues(typeof(AppColor)))
+            var testAccessor = AppSettings.GetTestAccessor();
+            string applicationExecutablePath = testAccessor.ApplicationExecutablePath;
+
+            try
+            {
+                testAccessor.ApplicationExecutablePath = Path.Combine(TestContext.CurrentContext.TestDirectory, "gitextensions.exe");
```

Comment:
why not use Setup / Teardown methods instead of try / finally ?

Reply:
We could, however doing so in `Setup`/`TearDown` would execute the code for all tests in the fixture, but it is only necessary for this test.

Reply:
It's also necessary for tests in `ThemeManagerTests`. The fact that some tests do not touch execution path when `ApplicationExecutablePath` is used looks to me like implementation detail which the test should not be aware of.

Reply:
obsolete